packages/jellyfish-api-core/__tests__/category/spv/listAnchorRewardConfirms.test..ts

```diff
@@ -0,0 +1,132 @@
+import { CreateAnchorResult } from '@defichain/jellyfish-api-core/category/spv'
```

Comment:
file name is `packages/jellyfish-api-core/__tests__/category/spv/listAnchorRewardConfirms.test..ts`
remove extra `.`